admin/templates/configuration/etc/init.d/xrootd

```diff
@@ -50,6 +50,21 @@ QSW_SCRATCHPATH="${QSERV_RUN_DIR}/tmp"
 QSW_SCRATCHDB="qservScratch"
 export QSW_RESULTPATH="${XROOTD_RUN_DIR}/result"
 
+# MemMan configuration
+export QSW_MEMMAN="MemManReal"
+export QSW_MEMMAN_MB="1500"
+export QSW_MEMMAN_LOCATION="${QSERV_RUN_DIR}/var/lib/mysql"
+
+# Worker Scheduler configuration
+export QSW_THRDPOOLSZ="15"
+export QSW_GROUPSZ="10"
+export QSW_PRIORITYSLOW="3"
+export QSW_PRIORITYMED="2"
+export QSW_PRIORITYFAST="1"
+export QSW_RESERVESLOW="2"
+export QSW_RESERVEMED="2"
+export QSW_RESERVEFAST="2"
+
```

Comment:
Definitely, the code behind it could be better as well. I created DM-5209 because I thought it was beyond the scope of this ticket.